### PR TITLE
Add mechanism to stop playing in case of dispose or changing source

### LINKED ISSUE #4851: VideoJs Plugin Not Releasing Connections
### Short description

__We have been using VideoJs with the Ant Media WebRTC plugin and it plays videos well, but it does not appear to release or clean up connections.__

https://github.com/ant-media/videojs-webrtc-plugin

### Environment

* Operating system and version: Windows 10 (Client) Ubuntu (Server)
* Java version: ?
* Ant Media Server version: Enterprise 2.5.3
* Browser name and version: Chrome 110.0.5481.178 

### Steps to reproduce

1. Load VideoJs and the Ant Media plugin.
2. Play a WebRTC video using `player.src(...)`. We'll call this `Source 1`.
3. Using the same player instance, play a different WebRTC video using `player.src(...)`. We'll call this `Source 2`.

Another method:

1. Load VideoJs and the Ant Media plugin.
2. Play a WebRTC video using `player.src(...)`. We'll call this `Source 1`.
3. Call `player.dispose()`.

In either scenario, when looking at the number of WebRTC viewers in Ant Media, notice that the number of viewers for `Source 1` does not go down. It will only go down if the page reloads (F5).

### Expected behavior
__When the src changes or the player is disposed, any existing WebRTC connections are closed and cleaned up.__

### Actual behavior
__All previous connections are kept alive for the life of the player, even if they are no longer being used to play anything.__

### Example
__I'll work on providing a workking code sample/example to demonstrate this. Right now it's in a React app in a private repo, but I'll work on recreating it in a different environment.__

### PR DESCRIPTION
**Pull request by Void-Spectra**
https://github.com/ant-media/Ant-Media-Server/issues/4851

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -58,13 +65,13 @@ class WebRTCHandler {
 
     this.source.pcConfig = { iceServers: JSON.parse(source.iceServers) };
     this.source.mediaServerUrl = `${source.src.split('/').slice(0, 4).join('/')}/websocket`;
-    this.source.streamName = source.src.split('/')[4].split('.webrtc')[0];
+    window.videojsWebrtcPlayerStreamName = this.source.streamName = source.src.split('/')[4].split('.webrtc')[0];
```
Comment (mekya) [inline src/plugin.js]: Using global objects may cause problem if there is more than one player in the same page. Please don't use global objects

**Review by mekya — CHANGES_REQUESTED** [review]

File: src/plugin.js (hunk 1 above)
Comment (hikeeba) [reply src/plugin.js]: We have systems that have multiple players on the same page. It would be great to support that model.

Comment (Void-Spectra) [reply src/plugin.js]: Sure, I will refactor to support multiple player in a page.

Comment (hikeeba) [reply src/plugin.js]: Here are the changes I made to a local copy to get this to work for us: https://editor.mergely.com/V0lz0NRX/

Feel free to use that as you wish. Basically I added some handling to properly dispose of everything when the handler is called again.

**Review by hikeeba — COMMENTED** [review]
This looks good to me. Thank you for the updates.

**Review by mekya — APPROVED** [review]
Thank you @hikeeba @mustafaboleken

**Pull request merged**